docs/standard/containerized-lifecycle-architecture/what-is-docker.md

```diff
@@ -3,47 +3,75 @@ title: What is Docker?
 
-![](./media/image2.png)
+![Docker containers can run anywhere, on-premises in the customer datacenter, in an external service provider or in the cloud, on Azure.](./media/image2.png)
 
-Figure 1-2: Docker deploys containers at all layers of the hybrid cloud
+**Figure 1-2**. Docker deploys containers at all layers of the hybrid cloud
 
-Docker image containers can run natively on Linux and Windows. However, Windows images can run only on Windows hosts and Linux images can run only on Linux hosts, meaning a host server or a VM.
+Docker image containers can run natively on Linux and Windows. However, Windows images can run only on Windows hosts and Linux images can run on Linux hosts and Windows hosts (using a Hyper-V Linux VM, so far), where host means a server or a VM.
 
-Developers can use development environments on Windows, Linux, or macOS. On the development computer, the developer runs a Docker host to which Docker images are deployed, including the app and its dependencies. Developers who work on Linux or on the Mac use a Docker host that is Linux based, and they can create images only for Linux containers. (Developers working on the Mac can edit code or run the Docker command-line interface \[CLI\] from macOS, but, as of this writing, containers do not run directly on macOS.) Developers who work on Windows can create images for either Linux or Windows Containers.
+Developers can use development environments on Windows, Linux, or macOS. On the development computer, the developer runs a Docker host where Docker images are deployed, including the app and its dependencies. Developers who work on Linux or on the Mac use a Docker host that is Linux-based, and they can create images only for Linux containers. (Developers working on the Mac can edit code or run the Docker command-line interface (CLI) from macOS, but as of this writing, containers don't run directly on macOS.) Developers who work on Windows can create images for either Linux or Windows Containers.
 
 To host containers in development environments and provide additional developer tools, Docker ships [Docker Community Edition (CE)](https://www.docker.com/community-edition) for Windows or for macOS. These products install the necessary VM (the Docker host) to host the containers. Docker also makes available [Docker Enterprise Edition (EE)](https://www.docker.com/enterprise-edition), which is designed for enterprise development and is used by IT teams who build, ship, and run large business-critical applications in production.
 
-To run [Windows Containers](/virtualization/windowscontainers/about/), there are two types of runtimes:
+To run [Windows Containers](https://msdn.microsoft.com/virtualization/windowscontainers/about/about_overview), there are two types of runtimes:
```

Comment:
The previous link was correct. Please don't bring back msdn links. We're trying to get rid of them with this massive effort #2037.
On the e-book, you can use the full path: https://docs.microsoft.com/virtualization/windowscontainers/about/index

```suggestion
To run [Windows Containers](/virtualization/windowscontainers/about/), there are two types of runtimes:
```

Reply:
Ok, there were time I wasn't clear about the links and other where the just slipped 😔